docs/arch/adr-36.md

```diff
@@ -0,0 +1,26 @@
+# ADR 36: NextJS App base paths
+
+15 November 2021
+
+## Context
+
+When we launched the NextJS apps, a single domain could only host one NextJS app. We gave each app its own subdomain: `fe-content-pages.zooniverse.org` and `fe-projects.zooniverse.org` then proxied URLs from `www.zooniverse.org` to those domains. Next data requests, on the `/_next` URL, are proxied from `www.zooniverse.org/_next` to `fe-project.zooniverse.org/_next`. We deliberately broke the content pages app, in favour of supporting projects.
+
+[Next 9.5](https://nextjs.org/blog/next-9-5), in July 2020, added support for multiple apps running on the same domain via the [`basePath`](https://nextjs.org/blog/next-9-5#customizable-base-path) config setting.
+
+## Decision
+
+Set base paths of `/about` and `/projects` for the content pages app and project app respectively.
+https://github.com/zooniverse/front-end-monorepo/pull/2519
+
+## Consequences
+
```

Comment:
A big consequence is that data requests to `/_next` can be supported for both apps, where they’re not supported for content pages at the moment.

Reply:
```suggestion
 - data requests to `/_next` can now be supported for both apps, at `/about/_next` and `/projects/_next`. 
```